iqb/.htaccess

```diff
@@ -4,3 +4,4 @@ RewriteRule ^core(.*)? https://github.com/iqb-berlin/mdc-core [R=302,L]
 RewriteRule ^mdc-core/(.*)? https://iqb-berlin.github.io/mdc-core/w3id.org/iqb/mdc-core/$1 [R=302,L] 
 RewriteRule ^github(.*)? https://iqb-berlin.github.io/ [R=302,L]
 RewriteRule ^(.*) https://www.iqb.hu-berlin.de/ [R=302,L]
+RewriteRule ^verona-modules(.*)? https://raw.githubusercontent.com/verona-interfaces/metadata/master/verona_modules_vocab.jsonld [R=302,L]
```

Comment:
Move this up a line to be above the wildcard rule.  Otherwise it will never be used.

Reply:
Fixed it, thank you very much!